packages/modules-runtime/package.js

```diff
@@ -6,12 +6,13 @@ Package.describe({
   documentation: "README.md"
 });
 
-Npm.depends({
-  install: "0.13.0"
-});
+// TODO: Upgrade install version once pull request is accepted.
+// Npm.depends({
+//   install: "0.13.0"
+// });
 
 Package.onUse(function(api) {
-  api.addFiles(".npm/package/node_modules/install/install.js", [
+  api.addFiles("install-fork.js", [
```

Comment:
I may attempt to submit a separate PR at:
https://github.com/benjamn/install
but not sure if @benjamn will be interest in accepting it as the repository does not seem to be maintained anymore.

Reply:
This is also something to consider, as we have not got answer for him, we may need to just accept the ` install-fork.js file` and make it part of our repository.